src/CronExpression.php

```diff
@@ -144,9 +144,9 @@ private function checkTime(string $time, string $format): bool
 
         $currentTime = $this->testTime->format($format);
 
-        // Handle repeating times (i.e. /5 for every 5 minutes)
-        if (strpos($time, '/') === 0) {
-            $period = substr($time, 1);
+        // Handle repeating times (i.e. /5 or */5 for every 5 minutes)
+        if (strpos($time, '/')) {
```

Comment:
This will fail when `/` is the first character because `0` is falsy. Try this:
```suggestion
        if (strpos($time, '/') !== false) {
```

Reply:
Thats fine, is the "/5" without the star "*/5" even a legit cron, out of curiosity? I've never heard of it before.